modules/ROOT/pages/design-create-publish-api-fragment.adoc

```diff
@@ -30,19 +31,16 @@ An API fragment is a RAML document that has a version and an identifier, but is
 == Procedure
 
 . On the *Projects* page in Design Center, click *Create* and select *Create Fragment*.
-. In the *New API Fragment* dialog, type a name for the project and select the type of API fragment that you want to create. Then, click *Create*.
+. In the *New Fragment* dialog, type a name for the project. You can change the name (but not the extension) later, if you want to.
+. Select the type of API fragment that you want to create. Then, click *Create Fragment*.
 +
 _Result:_ The text editor opens. The editor is divided into three panels:
```

Comment:
Do we need this "Result:" internal heading? If we follow the step with this text, it is pretty clear that it is the result of the step, without this extra element.